src/ui/Button/Button.tsx

```diff
@@ -92,6 +92,18 @@ export const Button = styled('button', {
       },
     },
     variant: {
+      claimFunds: {
```

Comment:
i think this should be defined in Claims rather than be a Button variant--the name suggests to me that it isn't that reusable